test/editions/PublishedIssueTest.scala

```diff
@@ -31,6 +31,7 @@ class PublishedIssueTest extends FreeSpec with Matchers {
         Some(true),
         Some(true),
         Some("byline"),
+        None,
```

Comment:
This test is for when the furniture is populated in order to ensure that it gets through to Alex. Can you please update to ensure a value is passed through?

Reply:
Done!